ftw/zipextract/tests/test_zipextract.py

```diff
@@ -22,6 +24,29 @@ def add_multi_zip_file(self):
             .attach_file_containing(self.asset('multi.zip'), u'multi.zip')
             .within(self.folder))
 
+    def add_zip_file_with_wrong_extension(self):
+        """ This is a normal zip file but with a wrong fileextension
+        """
+        self.file = create(
```

Comment:
I don't get why you create the file in an instance variable and return it. But for the sake of consistency let's leave it like that.